pkg/api/v1/exposure_types.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
```

Comment:
let's debate if this should be v1beta1 or v1

Reply:
I have no strong opinions.

Reply:
@whaught - you've got lots of API versioning experience :)

Reply:
It doesn't strongly matter to me, but the difference between calling this a beta and the v1 is usually more of a support, stability, and user expectation issue. Will early apps get a beta label? I'm guessing no: this project will need to build user trust early and may not have the long runway that some of the bigger apis get. I'd go with v1 and rev from there